Resources/Prototypes/_EE/Entities/Objects/Weapons/Melee/terminus.yml

```
@@ -0,0 +1,52 @@
+- type: entity
+  name: DT-8 "Terminus"
+  parent: BaseItem
+  id: Terminus
+  description: An advanced weapon crafted specifically for an oni. Capable of firing a spread of disabler shots from its tip.
+  components:
+  - type: OniOnly
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)